pmda/simple.py

```diff
@@ -0,0 +1,126 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+"""
+Parallel Analysis helper --- :mod:`pmda.simple`
+===============================================
+
+Collection of decorators to analyze
+
```

Comment:
It would be very useful to have a bit more text here at the top that explains the problem that is being solved and shows an example, perhaps the same as in the API docs below with a bit more explanation.

I think we want a user to look at the page and think "gee, that *is* simple, I will try it out."

Reply:
I think I would rather link to the classes. I don't like having to maintain example at two places. It's a bigger chance for me to forget to update them in the future.